README.md

```diff
@@ -1 +1 @@
-# git-study
\ No newline at end of file
+# git-study fork and pull request test
```

Comment:
Cool stuff!

Reply:
Feel free for back the message to the original form aiming at make possible to newbies use it for the same reason...